build.py

```diff
@@ -221,6 +250,34 @@ def _validate_android_args(args: argparse.Namespace):
             args.skip_csharp = True
 
 
+def _validate_ios_args(args: argparse.Namespace):
+    if args.ios:
+        if not util.is_mac():
+            raise ValueError("A Mac host is required to build for iOS")
+    
+        needed_args = [
+            args.ios_sysroot,
+            args.ios_arch,
+            args.ios_deployment_target,
+        ]
+        arg_names = [
+            "--ios_sysroot          "
+            + "<the location or name of the macOS platform SDK>",
+            "--ios_arch             "
+            + "<the Target specific architectures for iOS>",
+            "--ios_deployment_target  "
+            + "<the minimum version of the target platform>",
```

Comment:
nit: I don't think the line breaks are required and it makes it harder to parse that this is 1:1 with the needed_args list.

```suggestion
            "--ios_sysroot           <the location or name of the macOS platform SDK>",
            "--ios_arch              <the Target specific architectures for iOS>",
            "--ios_deployment_target <the minimum version of the target platform>",
```